_data/students.yml

```diff
@@ -146,13 +146,22 @@
   twitter: Adarsh_Kumar1
   bio: 16, Student Programmer, Web developer, Computer enthusiast
 
+
+- name: Ainesh Sootha
+  github: champforevaa
+  image: ainesh.jpg
+  twitter: ainesh_sootha
+  facebook: ainesh.sootha
+  bio: 15, Web developer, Game developer. Programmer, Indian 
```

Comment:
Remove the trailing whitespace after `bio: 15, Web developer, Game developer. Programmer, Indian` please to make the TravisCI build pass :)